pkgs/top-level/python-packages.nix

```diff
@@ -14606,7 +14606,16 @@ self: super: with self; {
     libsoxr = pkgs.soxr;
   };
 
-  spacy = callPackage ../development/python-modules/spacy { };
+  spacy = callPackage ../development/python-modules/spacy {
+    cython_0 = cython_0.overridePythonAttrs (old: rec {
```

Comment:
Could you leave a comment why we need to override cython in the code as well?

Reply:
Sure, thanks for the suggestion.